cmd/subcmds/output.go

```diff
@@ -45,28 +48,49 @@ func writeOutput(args *inArgs, collection ir.Collection, defs *ir.ConfigDefs) er
 		if args.prefix != "" {
 			args.outputFile = args.outputDir + "/" + args.prefix + "_" + suffix
 		}
-		if err := writeToFile(args, collection, vpc); err != nil {
+		if data, err = writeCollection(args, collection, vpc); err != nil {
+			return err
+		}
+		if err := writeToFile(args.outputFile, data); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func writeToFile(args *inArgs, collection ir.Collection, vpc string) error {
+func checks(args *inArgs, collection ir.Collection, defs *ir.ConfigDefs) error {
+	if err := updateFormat(args); err != nil {
+		return err
+	}
+	if args.locals && args.outputFmt == tfOutputFormat && (args.outputDir != "" || args.outputFile != "") {
+		if err := writeLocals(args, collection, defs); err != nil {
```

Comment:
This is not really a check, is it?
Here you should only check that if `args.locals` is `true`, then `args.outputFmt == tfOutputFormat`, and return an error if not.

Reply:
if `args.outputDir == "" && args.outputFile == ""`,  the collection will be written to stdout. Do we want to generate a `locals.tf` file in this case?

Reply:
Yes, we need to. Maybe in this case we should print the "locals" section at the beginning of the generated text rather than creating a file.